components/ErrorBoundary.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2021 The Rode Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import styles from "styles/modules/Errors.module.scss";
+import Button from "./Button";
+import { withRouter } from "next/router";
+
+class ErrorBoundary extends React.Component {
```

Comment:
So with the addition of this component can we remove this catch. https://github.com/rode/rode-ui/blob/main/pages/api/resources.js#L56

Reply:
I think technically we can, but keeping it there will stop the app from showing the error screen if something goes wrong that we can recover from on the client side.